electron-app/tests/e2e/specs/accounts-balances.spec.ts

```diff
@@ -54,10 +59,30 @@ describe('Accounts', () => {
       page.showsCurrency('EUR');
     });
 
-    it('add second entry', () => {
+    it('add second & third entires', () => {
       page.addBalance(manualBalances[1]);
       page.visibleEntries(2);
       page.isVisible(1, manualBalances[1]);
+      page.addBalance(manualBalances[2]);
+      page.visibleEntries(3);
+      page.isVisible(2, manualBalances[2]);
+    });
+
+    it('data is reflected in dashboard', () => {
+      page.getLocationBalances().then($manualBalances => {
```

Comment:
Is there a way we can write all of these without these `.then` callback function?

Reply:
No, this is apparently just how it's done in Cypress (https://docs.cypress.io/guides/core-concepts/variables-and-aliases.html). Because everything happens async in Cypress if you try to do it the normal way (e.g. just have a function that returns something then assign it to a const), it won't work. This was very painful to figure out, but it's possible I made it over-complicated. Let me know if you find a better way.